modules/update-service-install-cli.adoc

```diff
@@ -1,15 +1,15 @@
 [id="update-service-install-cli_{context}"]
 = Installing the OpenShift Update Service Operator by using the CLI
 
-You can use the {product-title} CLI to install the OpenShift Update Service Operator.
+You can use the OpenShift CLI (`oc`) to install the OpenShift Update Service Operator.
 
 .Procedure
 
 To install the OpenShift Update Service Operator by using the {product-title} CLI:
```

Comment:
Do you mind updating this instance of "{product-title} CLI" as well?

Reply:
Good catch! Resolved.